explanation/introduction-to.rst

```diff
@@ -64,3 +68,4 @@ help you navigate to the guides and other materials of most interest to you.
     Backups <intro-to/backups>
     Databases <intro-to/databases>
     High Availability <intro-to/high-availability>
+    eBPF <intro-to/ebpf>
```

Comment:
This doesn't feel like the right place for it -- I'm leaning more towards putting it alongside the virtualisation content. Wdyt?

Reply:
When I mention eBPF is a VM within the kernel, it is because eBPF is capable of running bytecode loaded from the user-space here. eBPF is a kernel resource to run code in the kernel without recompiling it or loading new modules. I am not sure if this should go alongside virtualisation... I did go though all the topics in the list here, and I am unsure if it belongs in any of them.

Reply:
Understood - in that case I agree, let's leave it as a standalone topic. We can always move it in the future if we add more content or a better grouping turns up.